assets/src/scripts/web-services/groundwater-levels.test.js

```diff
@@ -20,30 +21,93 @@ describe('web-services/groundwater-levels', () => {
         restoreConsole();
     });
 
+    describe('getGroundwaterServiceURL', () => {
+        it('Expects site no and format parameters but no others', () => {
+            const result = getGroundwaterServiceURL({
+                siteno: '11112222',
+                format: 'rdb'
+            });
+            expect(result).toContain('sites=11112222');
```

Comment:
This follows the same pattern as the tests in hydrograph/download-data.test.js. Allows order to change in the function implementation without breaking the tests.

Reply:
Much less brittle.